nemo/collections/asr/models/label_models.py

```diff
@@ -342,7 +355,6 @@ def forward_for_export(self, audio_signal, length):
         logits, embs = self.decoder(encoder_output=encoded, length=length)
         return logits, embs
 
-    @typecheck()
```

Comment:
remove typecheck to allow using multi-layer feature aggregation from a pretrained FC encoder, which is done by replacing the LabelModel's preprocessor